ciao-launcher/main.go

```diff
@@ -356,6 +353,28 @@ DONE:
 			if err != nil {
 				break DONE
 			}
+			clusterConfig,err := client.conn.ClusterConfiguration()
+			if err != nil {
+				glog.Errorf("Unable to get Cluster Configuration %v", err)
+				dialCh <- err
+				break DONE
+			}
+			computeNet = clusterConfig.Configure.Launcher.ComputeNetwork
+			mgmtNet = clusterConfig.Configure.Launcher.ManagementNetwork
+			diskLimit = clusterConfig.Configure.Launcher.DiskLimit
+			memLimit = clusterConfig.Configure.Launcher.MemoryLimit
+			if networking.Enabled() {
+				ctx, _ := context.WithCancel(context.Background())
+				ch := initNetworking(ctx)
+				err := <-ch
+				if err != nil {
+					glog.Errorf("Failed to init network: %v\n", err)
+					break DONE
+				}
+			}
+			defer shutdownNetwork()
+
+			ovsCh = startOverseer(&wg, client)
```

Comment:
14 is fine but I'm getting 16.  Have you rebased your patch set against the latest version of master?  If you have and you are still getting 14, perhaps I merged your patch incorrectly to my local workspace.  In any case, if there really is a problem here, Travis CI will fail the build when you update your PR.